remediation_worker/jobs/rds_remove_public_endpoint/rds_remove_public_endpoint.py

```diff
@@ -0,0 +1,156 @@
+# Copyright (c) 2021 VMware Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import annotations
+
+import json
+import logging
+import sys
+
+import boto3
+from botocore.exceptions import ClientError
+
+logging.basicConfig(level=logging.INFO)
+
+
+class RDSRemovePublicEndpoint():
+    def parse(self, payload):
+        """Parse payload received from Remediation Service.
+
+        Args:
+            payload: (str) JSON string containing parameters
+                    received from the remediation service.
+
+        Raises:
+            Exception: JSONDecodeError
+
+        Returns:
+            (dict) Dictionary of parsed parameters.
+        """
+        remediation_entry = json.loads(payload)
+        notification_info = remediation_entry.get("notificationInfo", None)
+        finding_info = notification_info.get("FindingInfo", None)
+
+        instance_id = finding_info.get("ObjectId", None)
+        region = finding_info.get("Region", None)
+
+        if instance_id is None:
+            logging.error("Missing parameters for 'INSTANCE_ID'.")
+            raise Exception("Missing parameters for 'INSTANCE_ID'.")
+
+        if region is None:
+            logging.error("Missing parameters for 'REGION'.")
+            raise Exception("Missing parameters for 'REGION'.")
+
+        logging.info("parsed params")
+        logging.info("  instance_id: %s", instance_id)
+        logging.info("  region: %s", region)
+
+        return {
+            "instance_id": instance_id,
+            "region": region
+        }
+
+    def remediate(self, client, instance_id):
+        """
+        Removes public access from RDS instances
+        and makes the instances private.
+
+        Args:
+            instance_id ([str]): The ID of the RDS instance.
+            client: Instance of the AWS boto3 client.
+        """
+
+        logging.info("Beginning RDS public endpoint evaluation.")
+
+        # Check to see if the database
+        # is publicly accessible
```

Comment:
Add logging messages

Reply:
logging.info("Beginning RDS public endpoint evaluation.")

        # Check to see if the database
        # is publicly accessible
        logging.info("executing client.describe_db_instances")
        logging.info(f"RDS instance={instance_id}")